charts/nri-kube-events/values.yaml

```diff
@@ -85,40 +34,80 @@ resources:
   #   memory: 128Mi
 
 rbac:
-  # Specifies whether RBAC resources should be created
+  # -- Specifies whether RBAC resources should be created
   create: true
 
+# -- Settings controlling ServiceAccount creation
+# @default -- See `values.yaml`
 serviceAccount:
-  # Specifies whether a ServiceAccount should be created
-  create: true
-  # The name of the ServiceAccount to use.
-  # If not set and create is true, a name is generated using the name template
+  # serviceAccount.create -- (bool) Specifies whether a ServiceAccount should be created
+  # @default -- `true`
+  create:
+  # If not set and create is true, a name is generated using the fullname template
   name: ""
   # Specify any annotations to add to the ServiceAccount
   annotations:
 
-# If you wish to provide additional annotations to apply to the pod(s), specify
-# them here
-# podAnnotations:
+# -- Annotations to add to the pod.
+podAnnotations: {}
+deployment:
+  # deployment.annotations -- Annotations to add to the Deployment.
+  annotations: {}
+# -- Additional labels for chart pods
+podLabels: {}
+# -- Additional labels for chart objects
+labels: {}
 
-# Pod scheduling priority
-# Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
-# priorityClassName: high-priority
+# -- Amount of time to wait until timeout to send metrics to the metric forwarder
+agentHTTPTimeout: "30s"
 
-nodeSelector: {}
+# -- Configure where will the metrics be writen. Mostly for debugging purposes.
+# @default -- See `values.yaml`
+sinks:
+  # -- Enable the stdout sink to also see all events in the logs.
+  stdout: false
+  # -- The newRelicInfra sink sends all events to New relic.
+  newRelicInfra: true
 
-tolerations: []
 
+# These values are managed by the common-library
```

Comment:
I would not create a section for the common library, I fear that in more complex chart it could be confusing to have them separated.

The user at the end does not care that much how options have been implemented

Reply:
You are completely right. Solved in [#0c95956](https://github.com/newrelic/nri-kube-events/pull/112/commits/0c95956e66de93abebab8828e9777d67540977f8)